package.json

```diff
@@ -45,7 +44,7 @@
     "eslint-plugin-prettier": "5.1.3",
     "eslint-plugin-require-extensions": "0.1.3",
     "prettier": "3.1.1",
-    "turbo": "1.11.3",
+    "turbo": "2.0.6-canary.0",
```

Comment:
This is only version of turbo that supports PNPM 9

Reply:
Wow really? Not the `2.0.9` supporting it as I see it's the latest? @segunadebayo